packages/media-sound/audacious-plugins/audacious-plugins.exlib

```diff
@@ -177,10 +186,10 @@ audacious-plugins_src_configure() {
         $(option_enable fluidsynth amidiplug)     # MIDI
         $(option_enable modplug)                  # Module decoder
         $(option_enable mp3 mpg123)               # MPEG-1 Layer I/II/III
+        $(option_enable opus)                     # Opus
         $(option_enable sndfile)                  # sndfile extensions
         $(option_enable vorbis)                   # Ogg Vorbis
         $(option_enable wavpack)                  # WavPack 4.31+
-        $(option_with ffmpeg ffmpeg ffmpeg)       # FFaudio
```

Comment:
Removing the configure check completely probably results in ffmpeg support being automagic, looks like it moved to --enable/--disable with https://github.com/audacious-media-player/audacious-plugins/commit/b7d7c94d1ca8502714a8e3679f94ce6d025e7ee4.

```
-$(option_with ffmpeg ffmpeg ffmpeg)       # FFaudio
+$(option_enable ffmpeg ffaudio)           # FFaudio
```